protocol/x/clob/rate_limit/order_rate_limiter.go

```diff
@@ -124,12 +127,20 @@ func (r *placeOrderRateLimiter) RateLimit(ctx sdk.Context, msg *types.MsgPlaceOr
 			1,
 			msg.Order.GetOrderLabels(),
 		)
-		r.rateLimitedSubaccounts[msg.Order.GetSubaccountId()] = true
+		r.rateLimitedAccounts[msg.Order.OrderId.SubaccountId.Owner] = true
+		r.rateLimitedSubaccounts[msg.Order.OrderId.SubaccountId] = true
```

Comment:
is this necessarily the desired behavior to do both? Shouldn't it be an if/else depending on if the ordertype is stateful or not?

Reply:
The purpose here was just to know at what number of accounts/subaccounts are hitting rate limits and thus it makes sense at a high level here to `AND` them.

Separately we could add different metrics to see which rate limit is being hit and at what level inside the rate limiter implementation itself.